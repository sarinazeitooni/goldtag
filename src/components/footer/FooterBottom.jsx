import React from 'react';
import Divider from "../divider/Divider";
import footerTexts from "./Data/footerData";
import useViewPort from "../../customIsMobileHook/useViewPort";
function FooterBottom(){
    const isMobile = useViewPort();
    return(
        <div className='footer-bottom-container'>
            <Divider className='footer-bottom-divider'/>
            <div className='footer-bottom'>
                <div>
                    <span>{footerTexts.copyRight}</span>
                    <div className='footer-bottom-text'>{footerTexts.rightText}</div>
                </div>
                    {!isMobile ? <div className='left-footer'>
                        {
                            footerTexts.leftFooter.map((item)=>{
                                return(
                                    <div key={item}>
                                        {item}
                                    </div>
                                )
                            })
                        }
                    </div> : ' '}
            </div>
        </div>
    )
};
export default FooterBottom;