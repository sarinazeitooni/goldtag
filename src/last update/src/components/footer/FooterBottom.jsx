import React from 'react';
import Divider from "../divider/Divider";
import {BrowserView} from "react-device-detect";
import footerTexts from "./Data/footerData";
function FooterBottom(){
    return(
        <div className='footer-bottom-container'>
            <Divider className='footer-bottom-divider'/>
            <div className='footer-bottom'>
                <div>
                    <span>{footerTexts.copyRight}</span>
                    <div className='footer-bottom-text'>{footerTexts.rightText}</div>
                </div>
                <BrowserView>
                    <div className='left-footer'>
                        {
                            footerTexts.leftFooter.map((item)=>{
                            return(
                                <div>
                                    {item}
                                </div>
                            )
                            })
                        }
                    </div>
                </BrowserView>
            </div>
        </div>
    )
};
export default FooterBottom;