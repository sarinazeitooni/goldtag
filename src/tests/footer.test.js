import React from 'react';
import {shallow} from 'enzyme';
import Footer from "../components/footer/footer";
import FooterBottom from "../components/footer/FooterBottom";
describe("footer component" , ()=>{
    it("footer should match snap shot" , ()=>{
        const wrapper = shallow(<Footer/>);
        expect(wrapper).toMatchSnapshot();
    });
    it(" footer bottom should match snap shot" , ()=>{
        const wrapper = shallow(<FooterBottom/>);
        expect(wrapper).toMatchSnapshot();
    });
    it(" footer should render 4 item section" , ()=>{
        const wrapper = shallow(<Footer/>);
        const sections = wrapper.find("div.footer-sections");
        expect(sections.length).toEqual(4);
    });
    it("footer bottom should render container" , ()=>{
        const wrapper = shallow(<FooterBottom/>);
        const conatiner = wrapper.find('div.footer-bottom-container');
        expect(conatiner.exists()).toBe(true);
    })
})