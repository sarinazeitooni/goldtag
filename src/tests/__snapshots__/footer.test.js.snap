// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer component  footer bottom should match snap shot 1`] = `ShallowWrapper {}`;

exports[`footer component footer should match snap shot 1`] = `ShallowWrapper {}`;
